custom_components/hwam_stove/translations/de.json

```diff
@@ -0,0 +1,202 @@
+{
+  "config": {
+    "step": {
+      "init": {
+        "data": {
+          "name": "Name",
+          "host": "Host",
+          "id": "ID"
+        }
+      }
+    },
+    "error": {
+      "already_configured": "Host bereits konfiguriet",
+      "id_exists": "ID existiert bereits",
+      "cannot_connect": "Kann nicht mit Host verbinden"
+    }
+  },
+  "device": {
+    "hwam_remote_device": {
+      "name": "HWAM Raum Temperatur Sensor"
+    },
+    "hwam_stove_device": {
+      "name": "HWAM Smart Ofen"
+    }
+  },
+  "entity": {
+    "binary_sensor": {
+      "refill_alarm": {
+        "name": "Nachlegen nötig",
+        "state": {
+          "off": "Nein",
+          "on": "Ja"
+        }
+      },
+      "maintenance_alarms": {
+        "name": "Wartung"
+      },
+      "maintenance_alarms_backup_battery_low": {
+        "name": "Backup Batterie"
+      },
+      "maintenance_alarms_o2_sensor_fault": {
+        "name": "O2 sensor"
+      },
+      "maintenance_alarms_o2_sensor_offset": {
+        "name": "O2 sensor offset"

```

Comment:
...and here?

Reply:
Also here, `Sauerstoff` would have my preference.
`offset` was translated to `correctiewaarde` (`correction value`) in the Dutch translation.